common/api-review/installations-exp.api.md

```diff
@@ -0,0 +1,34 @@
+## API Report File for "@firebase/installations-exp"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { FirebaseApp } from '@firebase/app-types-exp';
+import { FirebaseInstallations } from '@firebase/installations-types-exp';
+
+// @public (undocumented)

```

Comment:
@ChaoqunCHEN When you have a chance, can you add documentation for all APIs? It doesn't have to be part of this PR.

Reply:
sure.